docs/UserGuide.md

```diff
@@ -211,25 +211,26 @@ Adds student details to the person list.
 
 Format: `add id/STUDENT_ID n/NAME e/EMAIL g/GRADE [gp/GROUP_NAME]​...`
 
-* `NAME`, `STUDENT_ID`, `GRADE`, and `EMAIL` must be specified, `GROUP_NAME` is optional.
+* `NAME`, `STUDENT_ID`, `GRADE`, and `EMAIL` must be specified. `GROUP_NAME` is optional.
 * `NAME` is case-sensitive.
-    * e.g. `John Doe is different from john doe`
-* `STUDENT_ID` should **start with A follow by 7 digits and ends with a letter [A-Z]**.
+    * e.g. `John Doe` is different from `john doe`
+* `STUDENT_ID` should start with 'A', followed by 7 digits, and end with any capital letter. 
     * e.g. `A0123459X`
-* `EMAIL` should be of the format **local-part@domain**. The local-part should only be in the **format of e/E follow by 7 digits from [0-9]**. The **domain name must end with a domain label u.nus.edu**.
+* `EMAIL` should be of the format `username@domain`. The `username` should start with **e**, followed by 7 digits. The `domain` should be **u.nus.edu**.
     * e.g. `e0123456@u.nus.edu`
 * `GRADE` should be one of the valid grades: **[A+, A, A-, B+, B, B-, C+, C, D+, D, F]**.
+* `GROUP_NAME` cannot be empty, and must only contain alphanumeric characters and space.
```

Comment:
since [gp/GROUP_NAME] is optional, i think GROUP_NAME can be empty

Reply:
Thanks for pointing it out. It should be fixed now